stl/inc/xutility

```diff
@@ -4341,8 +4379,9 @@ _BidIt2 copy_backward(_BidIt1 _First, _BidIt1 _Last, _BidIt2 _Dest) { // copy [_
     auto _UFirst      = _Get_unwrapped(_First);
     const auto _ULast = _Get_unwrapped(_Last);
     auto _UDest       = _Get_unwrapped_n(_Dest, -_Idl_distance<_BidIt1>(_UFirst, _ULast));
-    _Seek_wrapped(_Dest, _Copy_backward_unchecked(_UFirst, _ULast, _UDest,
-                             bool_constant<_Ptr_copy_cat<decltype(_UFirst), decltype(_UDest)>::_Trivially_copyable>{}));
+    _Seek_wrapped(
+        _Dest, _Copy_backward_unchecked(_UFirst, _ULast, _UDest,
+                   bool_constant<_Memcopy_is_safe<decltype(_UFirst), decltype(_UDest), _Memcopy_cat::_Copy>>{}));

```

Comment:
Being 3 lines now, this could benefit from `using`.

Reply:
Done